lib/internal/v8_prof_processor.js

```diff
@@ -12,7 +12,7 @@ const console = require('internal/console/global');
 const vm = require('vm');
 const { SourceTextModule } = require('internal/vm/module');
 
-const natives = internalBinding('natives');
+const { natives } = internalBinding('builtins');
```

Comment:
Do destructuring bindings come with a cost? Does this apply here as well?

Reply:
No, it's just sugar for `const natives = internalBinding('builtins').natives;`